### PR TITLE
Ringback in call connect sdk 62

### PR DESCRIPTION
**Pull request by kevingarabedian2**
Relates to https://github.com/signalwire/relay-apis/issues/62

### CODE REVIEW

**Comment by kevingarabedian2** [comment]
closes 162

File: signalwire-dotnet/Relay/Calling/CallDevice.cs [hunk 1]
```diff
@@ -26,6 +26,8 @@ public sealed class PhoneParams
 
             [JsonProperty("timeout", NullValueHandling = NullValueHandling.Ignore)]
             public int Timeout { get; set; } = 30;
+
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/Calling/CallDevice.cs]: Remove whitespace, static analyzer will complain

File: signalwire-dotnet/Relay/CallingAPI.cs [hunk 2]
```diff
@@ -382,7 +382,7 @@ private void OnCallingEvent_SendDigits(Client client, BroadcastParams broadcastP
 
             call.SendDigitsStateChangeHandler(callEventParams, sendDigitsParams);
         }
-
+
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/CallingAPI.cs]: Remove leading whitespace on empty lines, static analyzer won't like this

File: signalwire-dotnet/Relay/CallingAPI.cs [hunk 3]
```diff
@@ -46,16 +46,16 @@ public PhoneCall NewPhoneCall(string to, string from, int timeout = 30)
...
             mCalls.TryAdd(call.TemporaryID, call);
             OnCallCreated?.Invoke(this, call);
             return call;
         }
 
-        public DialResult DialPhone(string to, string from, int timeout = 30) { return NewPhoneCall(to, from, timeout).Dial(); }
+        public DialResult DialPhone(string to, string from, int timeout = 30 ) { return NewPhoneCall(to, from, timeout).Dial(); }
 
-        public DialAction DialPhoneAsync(string to, string from, int timeout = 30) { return NewPhoneCall(to, from, timeout).DialAsync(); }
+        public DialAction DialPhoneAsync(string to, string from, int timeout = 30 ) { return NewPhoneCall(to, from, timeout).DialAsync(); }
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/CallingAPI.cs]: Remove the spaces that were added after the default parameters

File: signalwire-dotnet/Relay/Calling/LL_BeginParams.cs [hunk 4]
```diff
@@ -10,9 +10,12 @@ namespace SignalWire.Relay.Calling
     public sealed class LL_BeginParams
     {
         [JsonProperty("device", Required = Required.Always)]
-        public CallDevice Device { get; set; }
+        public CallDevice Device { get; set; }
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/Calling/LL_BeginParams.cs]: More trailing whitespace, analyzer won't like

File: signalwire-dotnet/Relay/Calling/Call.cs [hunk 5]
```diff
@@ -1850,8 +1852,8 @@ protected override async Task<DialResult> InternalDialAsync()
                             Timeout = Timeout,
                         },
                     },
-                    TemporaryCallID = mTemporaryID,
-                });
+                    TemporaryCallID = mTemporaryID
+                });
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/Calling/Call.cs]: Trailing whitespace, static analyzer won't like this

File: signalwire-dotnet/Relay/Calling/LL_BeginParams.cs [hunk 6]
```diff
@@ -10,9 +10,12 @@ namespace SignalWire.Relay.Calling
...
     {
         [JsonProperty("device", Required = Required.Always)]
-        public CallDevice Device { get; set; }
+        public CallDevice Device { get; set; } 
 
         [JsonProperty("tag", NullValueHandling = NullValueHandling.Ignore)]
         public string TemporaryCallID { get; set; }
+
+        [JsonProperty("ringback", NullValueHandling = NullValueHandling.Ignore)]
+        public CallRingback Ringback { get; set; }
```
Comment (Astaelan) [inline signalwire-dotnet/Relay/Calling/LL_BeginParams.cs]: Remove this, not part of the spec yet

**Comment by Astaelan** [comment]
Ringback was added to Begin and removed from Connect, that's backwards. Remove from Begin and add it to Connect.

**Pull request merged**